docs/en/sql-reference/syntax.md

```diff
@@ -128,6 +128,50 @@ Result:
 └────────────────────────────┘
 ```
 
+## Defining and Using Query Parameters
+
+Query parameters can be defined using the syntax `param_name=value`, where `name` is the name of the parameter. Parameters can by defined using the `SET` command, or from the command-line using `--param`.
+
+To retrieve a query parameter, you specify the name of the parameter along with its data type surrounded by curly braces:
```

Comment:
no longer needed, this will also work: `SET param_d = {'10': [11, 12], '13': [14, 15]};`